src/components/ExpensifyWordmark.js

```diff
@@ -9,13 +10,23 @@ import withEnvironment, {environmentPropTypes} from './withEnvironment';
 import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
 import compose from '../libs/compose';
 import themeColors from '../styles/themes/default';
-import styles from '../styles/styles';
 import * as StyleUtils from '../styles/StyleUtils';
 import variables from '../styles/variables';
 
 const propTypes = {
     ...environmentPropTypes,
     ...windowDimensionsPropTypes,
+
+    /** The styles to apply for the View wrapping the svg */
+    containerStyles: PropTypes.arrayOf(PropTypes.object),
```

Comment:
This is failing the linter but I see it's how we do this elsewhere in the code base so not sure if I should ignore this like we do here or if there's some other proptypes i should use? https://github.com/Expensify/App/blob/8f1617419ac34ba40bc4e4bfd4ed7fd1de0f12cd/src/components/FormAlertWithSubmitButton.js#L12-L14